docs/artifacts/aggregated-research-plugin-blueprint.html

```
@@ -0,0 +1,144 @@
+    <pre class="mermaid">
+%%{init: {'theme':'base','themeVariables':{'primaryColor':'#FFFFFF','primaryTextColor':'#0F1F24','primaryBorderColor':'#526168','lineColor':'#0F1F24','fontFamily':'DM Mono, monospace','fontSize':'12px'}}}%%
+flowchart TB
+  subgraph MP["ray-manaloto/claude-code-marketplace"]
+    PJ["aggregated-research/.claude-plugin/plugin.json<br/>NO fixed version → tracks the git SHA<br/>dependencies: firecrawl · exa · context7 · last30days"]
+    SK["skills/ — the spine + one per adapter<br/>each defers to `aggregated-research &lt;verb&gt; --help`"]
+    AG["agents/ — synthesist, adversarial verifier"]
+    HK["hooks/ SessionStart:<br/>fetch the pinned CLI release from GitHub<br/>into $CLAUDE_PLUGIN_DATA when the version differs;<br/>validated against the manifest schema's hooks shape"]
+    RU["rules/security.md — CLI output is file-isolated,<br/>read back with grep/head (firecrawl's rule, copied)"]
+  end
+  subgraph CLI["the CLI — installed SEPARATELY (uv tool / mise pipx pin)"]
+    C["aggregated-research"] --> T["trackers ✔ (kb_setup/research/trackers.py)"]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)